content/security/index.adoc

```diff
@@ -8,70 +8,76 @@ The Jenkins project takes security seriously.
-- link:https://feeds.feedburner.com/jenkins-security-advisories[`jenkinsci-advisories` RSS feed]
+[IMPORTANT]
+====
+Even if you run Jenkins on a private network and trust everyone in your team, security issues in Jenkins can still impact you:
 
-Whenever a security update is released, we inform the `jenkinsci-advisories` and link:https://oss-security.openwall.org/wiki/mailing-lists/oss-security[`oss-security`] mailing lists.
+- https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSRF vulnerabilities] are a risk even if attackers have no direct access to Jenkins.
+- Does Jenkins build source code you haven't audited, using build scripts someone else wrote, displaying generated reports on its web UI? All of this is a potential a security concern.
+====
 
-IMPORTANT: We recommend you subscribe to link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[`jenkinsci-advisories`] even if your Jenkins instance is not publicly accessible to receive timely notifications on security updates.
 
 [[reporting-vulnerabilities]]
 == How to Report a Security Vulnerability
 
 If you find a vulnerability in Jenkins, please report it in the issue tracker under the link:https://issues.jenkins-ci.org/browse/SECURITY[SECURITY project].
-This project is configured in such a way that only the reporter and the security team can see the details.
-By restricting access to this potentially sensitive information, we can work on a fix and deliver it before the method of attack becomes well-known.
+This project is configured in such a way that only the reporter, the maintainers, and the Jenkins security team can see the details.
+Restricting access to this potentially sensitive information allows core and plugin maintainers to develop effective security fixes that are safe to apply.
 We provide issue reporting guidelines and an overview of our process on link:reporting[Reporting Security Vulnerabilities].
 
 If you are unable to report using our issue tracker, you can also send your report to the private Jenkins Security Team mailing list:
 `jenkinsci-cert@googlegroups.com`
 
-To show our appreciation for your help, we'll send you link:https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for privately reported, valid vulnerability reports.
+To show our appreciation for your help, we'll send you https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for privately reported, valid vulnerability reports.
 
-[[cna]]
-== CVE Numbers Authority
 
-The Jenkins project is a link:https://cve.mitre.org/[CVE Numbers Authority] (CNA) for Jenkins and Jenkins plugins published by the Jenkins project (listed on https://plugins.jenkins.io/[plugins.jenkins.io] and/or hosted in https://github.com/jenkinsci[the jenkinsci GitHub organization]).
-Contact us at `jenkinsci-cert@googlegroups.com` if you have questions about the Jenkins CNA.
+== Learn More
```

Comment:
I absolutely love the formatting of this section.  The layout is easy to read, the short descriptions of the terms are good for many readers, and the hyperlinks lead to much more detailed materials.  Thanks very much for such an elegant use of a definition list for navigation and concepts.